### PR TITLE
update tests and identifiers

### PR DESCRIPTION
**Pull request by dualcnhq**

### CODE REVIEW

File: config/index.ts [hunk 1]
```diff
@@ -4,11 +4,79 @@ dotenv.config();
...
+    password: process.env.PATRON_PASSWORD || '',
   },
 }
+
+export const identifier = {
+  about: {
+    url: '/about',
+    jobs: '/about/jobs.php',
+    terms: '/about/terms.php',
+    news: '/about/news-stories'
```
Comment (bfalling) [inline config/index.ts]: SUGGESTION: Add trailing comma.

File: config/index.ts [hunk 2]
```diff
@@ -4,11 +4,79 @@ dotenv.config();
...
+  },
+  av: {
+    url: '',
+    default: '/details/CSPAN_20160425_022500_2011_White_House_Correspondents_Dinner'
```
Comment (bfalling) [inline config/index.ts]: SUGGESTION: Add trailing comma.

File: config/index.ts [hunk 3]
```diff
@@ -4,11 +4,79 @@ dotenv.config();
...
+    default: '',
+  },
+  books: {
+    url: '/details',
+    default: '/details/theworksofplato01platiala',
+  },
+  lending: {
+    chromium: 'hitrun0000haro',
+    firefox: 'cornellstudiesin26unse',
+    webkit: 'annualreport0000meth_x3r2'
```
Comment (bfalling) [inline config/index.ts]: SUGGESTION: Add trailing comma.

File: package.json [hunk 4]
```diff
@@ -18,15 +18,14 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.45.0",
+    "@playwright/test": "^1.49.0",
     "@types/node": "^20.11.17",
-    "browserstack-node-sdk": "^1.34.18",
+    "browserstack-node-sdk": "^1.34.23",
```
Comment (bfalling) [inline package.json:23]: QUESTION: I saw you removed references to browserstack-node-sdk from the command-line above. Is the package still needed?

File: package.json [hunk 5]
```diff
@@ -18,15 +18,14 @@
...
     "dotenv": "^16.0.3",
     "eslint": "^8.44.0",
     "eslint-config-prettier": "^8.8.0",
     "husky": "^8.0.3",
     "minimist": "^1.2.8",
-    "playwright": "^1.45.0",
```
Comment (bfalling) [inline package.json:29]: QUESTION: No playwright??

File: tests/about/about.spec.ts [hunk 6]
```diff
@@ -1,7 +1,20 @@
...
+
+test.beforeEach(async ({ request, context }) => {
+  if(process.env.IS_REVIEW_APP === 'true') {
+    await context.addCookies([{
+      name: 'beta-access',
+      value: process.env.BETA_ACCESS_TOKEN || '',
+      path: '/',
+      domain: '.archive.org'
+    }]);
+  }
```
Comment (bfalling) [inline tests/about/about.spec.ts]: QUESTION: This clause is repeated in every test file, which would make it hard to change if, for example, we needed to adjust it; we'd need to change it in many places. How can we extract it (or parts of it) to some kind of mix-in or import or DRY it up in some other way?
```
  if(process.env.IS_REVIEW_APP === 'true') {
    await context.addCookies([{
      name: 'beta-access',
      value: process.env.BETA_ACCESS_TOKEN || '',
      path: '/',
      domain: '.archive.org'
    }]);
  }
```

File: tests/about/about.spec.ts [hunk 7]
```diff
@@ -25,18 +38,19 @@ test('About > Jobs page has correct title and text', async ({ page }) => {
...
   await expect(page).toHaveTitle(/Terms of Use/);
   await expect(page.locator('h1:has-text("Terms of Use")')).toBeVisible();
   await expect(page.locator('#maincontent')).toContainText(
     'This terms of use agreement',
   );
 });
 
-test('About > News Stories page has correct title and text', async ({
+// News Stories page is currently down
+test.skip('About > News Stories page has correct title and text', async ({
```
Comment (bfalling) [inline tests/about/about.spec.ts]: NOTE: It's now back up! :)

**Review by bfalling — COMMENTED** [review]
Great work on this. Some comments/questions/suggestions left.

Note that this PR is doing many things at once: switching to fixtures, adding cookies, adding new page tests, etc. It's probably too many things for one PR. You don't have to change this PR, but next time, please let each PR accomplish one thing, and split changes out into multiple PR's as appropriate. Thanks!

File: package.json (hunk 5 above)
Comment (dualcnhq) [reply package.json:29]: we just need @playwright/test for the playwright dependencies

File: package.json (hunk 4 above)
Comment (dualcnhq) [reply package.json:23]: yes, it's still needed in package.json line 15

File: tests/about/about.spec.ts (hunk 6 above)
Comment (dualcnhq) [reply tests/about/about.spec.ts]: this is now refactored

File: tests/about/about.spec.ts (hunk 7 above)
Comment (dualcnhq) [reply tests/about/about.spec.ts]: updated

**Pull request merged**